pkgs/development/cuda-modules/aliases.nix

```diff
@@ -1,4 +1,20 @@
 # Packges which have been deprecated or removed from cudaPackages
-final: prev: {
+final: prev:
+let
+  mkDeprecation =
+    oldName: newName: newPkg:
+    final.lib.warn "cudaPackages.${oldName} is deprecated, use ${newName} instead" newPkg;
```

Comment:
`mkRenamed`/`mkRenamedAttribute` (analogous to `mkRenamedOptionModule`) is probably clearer, especially if we later want to deprecate things (e.g. `cudatoolkit`) by actually removing them

Reply:
Sure, sounds good with me